pkg/defaulting/images.go

```diff
@@ -28,7 +28,14 @@ const (
 	// PublicECSContainerRegistry corresponds to the datadoghq PublicECSContainerRegistry registry
 	PublicECSContainerRegistry ContainerRegistry = "public.ecr.aws/datadog"
 	// DefaultImageRegistry corresponds to the datadoghq containers registry
-	DefaultImageRegistry = GCRContainerRegistry // TODO: this is also defined elsewhere and not used; consolidate
+	DefaultImageRegistry = "gcr.io/datadoghq"
+	// Default Image names
+	DefaultAgentImageName        string = "agent"
+	DefaultClusterAgentImageName string = "cluster-agent"
+	DefaultAzureImageRegistry    string = "datadoghq.azurecr.io"
+	DefaultEuropeImageRegistry   string = "eu.gcr.io/datadoghq"
+	DefaultAsiaImageRegistry     string = "asia.gcr.io/datadoghq"
+	DefaultGovImageRegistry      string = "public.ecr.aws/datadog"
 	// JMXTagSuffix prefix tag for agent JMX images
 	JMXTagSuffix      = "-jmx"
 	AgentDevImageName = "datadog/agent-dev"
```

Comment:
```suggestion
	DefaultImageRegistry = "gcr.io/datadoghq"
	DefaultAzureImageRegistry    string = "datadoghq.azurecr.io"
	DefaultEuropeImageRegistry   string = "eu.gcr.io/datadoghq"
	DefaultAsiaImageRegistry     string = "asia.gcr.io/datadoghq"
	DefaultGovImageRegistry      string = "public.ecr.aws/datadog"
	// JMXTagSuffix prefix tag for agent JMX images
	JMXTagSuffix      = "-jmx"
	// Default Image names
	DefaultAgentImageName        string = "agent"
	DefaultClusterAgentImageName string = "cluster-agent"
	AgentDevImageName = "datadog/agent-dev"
```
It's a re-ordering (indent is probably incorrect, done in the Github UI), but we could separate the image names from the registries + we also redefine the same constants lower in private https://github.com/DataDog/datadog-operator/pull/1639/files#diff-936c9edb7068ecd448ca5aed9ed6b806ad5c716cf1d5ecfb6c71de8a5b0aa28aL38-L39 should we remove them to rely on the public consts `DefaultAgentImageName` and `DefaultClusterAgentImageName` ?

Reply:
Thanks for catching that! Reordered and deleted the private variables.